sql-schema/269.sql

```diff
@@ -1 +1 @@
-INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'apm_sessions',  'apm',  'Active Sessions',  0);
+INSERT INTO `graph_types`(`graph_type`, `graph_subtype`, `graph_section`, `graph_descr`, `graph_order`) VALUES ('device',  'bigip_apm_sessions',  'apm',  'Active Sessions',  0);
```

Comment:
We are already in 269 Schema, you cant modify it. You should create 270.sql.

Also, if you are modifiying a current existing graph type (changing apm_sessions to bigip_apm_sessions) you must do an UPDATE instead of INSERT